xls/dslx/stdlib/bfloat16.x

```diff
@@ -42,6 +42,9 @@ pub fn is_nan(f: BF16) -> u1 { apfloat::is_nan<u32:8, u32:7>(f) }
 pub fn is_zero_or_subnormal(f: BF16) -> u1 {
   apfloat::is_zero_or_subnormal<u32:8, u32:7>(f)
 }
+pub fn eq_2(x: BF16, y: BF16) -> u1 {
```

Comment:
Invented style nit: space between functions.

Reply:
Done.  Apply for invention disclosure?